recipes/libftdi/1.x/conanfile.py

```diff
@@ -79,7 +82,7 @@ def package(self):
         lib_folder = os.path.join(self.package_folder, "lib",)
         tools.rmdir(os.path.join(lib_folder, "cmake"))
         tools.rmdir(os.path.join(lib_folder, "pkgconfig"))
-        os.unlink(os.path.join(self.package_folder, "bin", "libftdi1-config"))
+        #os.unlink(os.path.join(self.package_folder, "bin", "libftdi1-config"))
```

Comment:
Is this an executable? Then just remove the line and add the typical lines in the `package_info` method, just in case it is needed by some other recipe as build-requires:

```python
        bindir = os.path.join(self.package_folder, "bin")
        self.output.info("Appending PATH environment variable: {}".format(bindir))
        self.env_info.PATH.append(bindir)
```

Reply:
That was just an obsolet line. I removed it